### PR TITLE
Cluster list actions

### LINKED ISSUE #707: Please add support for hibernate/wake-up as action in the clusters list
**What would you like to be added**:
It would be nice to have hibernate/wake-up as action in the clusters list. There I can find only the kubeconfig and deletion, but hibernate/wake-up I do more often.

Screenshot 2020-05-29 at 06 31 08

**Why is this needed**:
Convenience, no hard reason.

**Remark**
There is also https://github.com/gardener/dashboard/issues/631, but it's about mass operations. Still, I link it because it would require to have the operation available first. The users example was also about hibernation.

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
This PR adds the ability to trigger cluster actions directly from the cluster list without navigating to the cluster details page.

Screen Shot 2020-08-12 at 16 12 53

**Which issue(s) this PR fixes**:
Fixes #707

**Special notes for your reviewer**:

**Release note**:

```noteworthy user
Cluster actions can now be accessed directly from the cluster list without navigating to the cluster details page
```

### CODE REVIEW

**Comment by holgerkoser** [comment]
According to [Material Design](https://material.io/components/menus#anatomy) a Text and icon list menu should look like this
image
Icon on the left, text not uppercase, no headline in the menu.

**Comment by holgerkoser** [comment]
I would add at least one separator before `Delete Cluster`

File: frontend/src/components/DeleteCluster.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 
 <template>
-  <action-icon-dialog
+  <action-button-dialog
```
Comment (holgerkoser) [inline frontend/src/components/DeleteCluster.vue:18]: The name should be something like `action-dropdown-menu`. It is not a dialog and in the menu shouldn't occur buttons.

File: frontend/src/components/ShootListRowActions.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        <template v-slot:activator="{ on: tooltip }">
+          <v-btn v-on="{ ...menu, ...tooltip}" icon class="cyan--text text--darken-2">
+            <v-icon class="cursor-pointer">more_vert</v-icon>
+          </v-btn>
+        </template>
+        Cluster Actions
+      </v-tooltip>
+    </template>
+    <v-list subheader dense class="actionMenuItem" @click.native.capture="actionMenu=false">
+      <v-subheader>Cluster Actions</v-subheader>
```
Comment (holgerkoser) [inline frontend/src/components/ShootListRowActions.vue:30]: Remove subheader

File: frontend/src/components/DeleteCluster.vue (hunk 1 above)
Comment (holgerkoser) [inline frontend/src/components/DeleteCluster.vue:18]: Do not use this component in the menu or add more config options.

File: frontend/src/components/dialogs/ActionButtonDialog.vue [hunk 3]
```diff
@@ -16,23 +16,29 @@ limitations under the License.
 
 <template>
   <div v-if="canPatchShoots">
```
Comment (holgerkoser) [inline frontend/src/components/dialogs/ActionButtonDialog.vue:18]: I think it is strange to decide inside a component that it should not render something. This should be done outside even if it redundant. What makes sense inside the component would be to disable the action.

Comment (grolu) [reply frontend/src/components/dialogs/ActionButtonDialog.vue:18]: Okay but this has nothing to do with this PR. This is logic related to the member roles / permissions. I would suggest to change this when we introduce the uam role, which I will be working on next.

**Pull request merged**